### ISSUE TITLE
crashed when trying to click on "p" in the "transform" for a selected unit/element/instance

### ISSUE THREAD

**Issue by Whurrhurr** [simon-wh/PAYDAY-2-BeardLib-Editor#305]
Application has crashed: C++ exception
mods/BeardLib-Editor/Classes/MenuUtils.lua:258: attempt to index field 'override_panel' (a nil value)

end
local mouse_1 = Idstring("1")
Item.UNCLICKABLE = 1 --null is "unknown".

SCRIPT STACK

RunCallback() 975
MousePressed() 1098
MousePressed() 1124
MousePressed() 1124
MousePressed() 1124
MousePressed() 1124
mouse_press() 416
lib/managers/mousepointermanager.lua:339

-------------------------------

basically the "c" (copy) and "p" (paste) letters are inside the numbers or some shit, pressing them causes the game to crash with this
